packages/elvis/src/components/form/input.scss

```diff
@@ -9,6 +9,7 @@
   syntax: '<angle>';
 }
 
+.e-form-field--small .e-input,
```

Comment:
This "top part" of the styles contains defaults, so we should not mix in modifiers in this part of the styles.

Reply:
At the bottom of the file, there is a block containing alle the "small input" modifiers. Maybe it makes sense to add 
``` css
color: var(--e-color-text-disabled-1);
```
in there as a modifier when it is small _and_ disabled? 🙂

Reply:
Good point!